templates/cow/changelog/logs/audit_mode.md.twig

```diff
@@ -0,0 +1,26 @@
+{%- from "changelog/logs/macros.md.twig" import format_commit, print_section_by_type as print_section -%}
+
+{%- set SECTIONS = [
+    'Security',
+    'Features and Enhancements',
+    'Bugfixes',
+    'API Changes',
+    'Dependencies',
+    'Documentation'
+]
+-%}
```

Comment:
Could this be exposed as a PHP constant and shared with the logic there?

Reply:
Yes, I think we could. However, I deliberately left it in the template as a presentational detail. My thinking here was that the template should decide what sections to show or how to group them. Here we exclude the 'Maintenance' section, for example.

Reply:
Right, that does make sense as a separate list, then. Happy to leave it here, though it _might_ pay to add a comment indicating where the definitive list is located (e.g. if someone wanted to add/remove a section.)

Reply:
added some comments